src/navigation/navigation.html

```diff
@@ -9,13 +9,13 @@
     <nav id="nav-container">
 
         <!-- logo -->
-        <div class="logo">
+        <div class="logo-wrapper">
             <a href="https://binary.com" target="_blank">
-                <div>
-                    <img class="responsive" src="https://www.binary.com/images/pages/binary-symbol-logo.svg">
+                <div class="logo">
+                    <img class="responsive" src="<style-url>/images/logo/symbol.svg">
                 </div>
-                <div>
-                    <img class="responsive" src="https://www.binary.com/images/pages/binary-type-logo.svg" style="width:80%;">
+                <div class="binary-logo-text">
+                    <img class="responsive" src="<style-url>/images/logo/type.svg" style="width:80%;">
                 </div>
             </a>
         </div>
```

Comment:
@4p00rv can you also use this logo code in 404.html and any other place that is using it?